src/main/java/seedu/duke/operationlist/FlightList.java

```diff
@@ -99,14 +105,143 @@ private static boolean findAndRemoveFlight(String flightNumber) {
         boolean isFlightFound = false;
         assert !flights.isEmpty();
         for (FlightInfo flight : flights) {
-            if (flight.getFlightNum().equals(flightNumber)) {
+            if (flight.getFlightNumber().equals(flightNumber)) {
```

Comment:
Similar to the above, can refractor into a single Boolean for easier identification